docker-compose.latest.yml

```diff
@@ -1,6 +1,7 @@
 services:
   api_gateway:
     profiles:
+      - all-with-models
```

Comment:
`all-with-models` is not necessary here and it's not a component profile. We can just use the profile `all` and enable the init-model by the model-backend config directly.